app/frontend/actions.js

```diff
@@ -583,10 +583,11 @@ export default ({setState, getState}) => {
 
   const calculateMaxDonationAmount = async () => {
     const state = getState()
-    const maxDonationAmount = await wallet.getMaxDonationAmount(
-      state.sendAddress.fieldValue,
-      state.sendAmount.coins
-    )
+    const maxDonationAmount = await wallet
+      .getMaxDonationAmount(state.sendAddress.fieldValue, state.sendAmount.coins)
+      .catch((e) => {
+        handleError('sendAmountValidationError', {code: e.name})
+      })
```

Comment:
When testing I noticed the network error is unhandled in case of calculating max amount and donation amount, so in case a user has network issues or our servers are down during fee calculation, unexpected error modal shows up.